.github/workflows/statistician-dive.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Statistician docker check
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/statistician-dive.yml'
+      - 'statistician/**'
+
+  push:
+    paths:
+      - '.github/workflows/statistician-dive.yml'
+      - 'statistician/**'
+
+env:
+  ORG: odc
+  IMAGE: statistician 
```

Comment:
and `datacube-statistician` https://hub.docker.com/r/opendatacube/datacube-statistician

Reply:
comment as above.